tests/cell_reference/screen-output

```diff
@@ -34,6 +34,14 @@ Level: 1 Index: 2
 Level: 1 Index: 2
 Level: 1 Index: 3
 Level: 1 Index: 3
+Level: 1 Index: 0
```

Comment:
We now have the cell object in more places than before (because one of the constructors of MaterialModelInputs was changed to use the get_cell function). So I thought it made sense that there is more output now (the evaluate function outputs the cell if it gets it).